idea-plugin/src/main/kotlin/com/itangcent/idea/plugin/api/export/markdown/MarkdownFormatter.kt

```diff
@@ -235,9 +244,22 @@ class MarkdownFormatter {
                 handle("| name  |  type  |  desc  |\n")
                 handle("| ------------ | ------------ | ------------ |\n")
                 response.body?.let { parseBody(0, "", "", it, handle) }
+                // handler json example
+                handle("\n**Response Demo：**\n\n")
+                parseToJson(handle, request)
             }
+        }
 
+    }
+
+    private fun parseToJson(handle: (String) -> Unit, request: Request) {
```

Comment:
how about to define parseToJson as
```
    private fun parseToJson(handle: (String) -> Unit, body: Any?) {
        handle("```json\n")
        body?.let {
            if (it != 0) {
                handle(RequestUtils.parseRawBody(it))
            }
        }
        handle("\n```\n")
    }
```
then call it by 
`parseToJson(request.body)` or `parseToJson(response.body)`